backend/Dockerfile

```diff
@@ -9,8 +9,5 @@ COPY tsconfig.json ./
 RUN npm install
 COPY . .
 RUN npm run build
-ENV PORT=8080
-ENV DB_URI=${DB_URI}
```

Comment:
Maybe we should add a build stage for production if these lines are necessary?

Reply:
Whatever is more elegant and understandable 🤓